docs/configuration/promtool.md

```diff
@@ -0,0 +1,179 @@
+---
+title: HTTP Configuration for promtool
+sort_rank: 6
+---
+
+# HTTP Configuration File for promtool
+
+Promtool is a versatile CLI tool for Prometheus that supports validation, debugging, querying, unit testing, tsdb management, pushing data and experimental PromQL editing.
+
+Prometheus supports basic authentication and TLS and since promtool needs to connect to Prometheus, we need to provide the authentication details. To specify those authentication details, use the `--http.config.file` for all requests that need to communicate with Prometheus.
+For instance, if you would like to check whether your local Prometheus server is healthy, you would use:
+```bash
+promtool check healthy --url=http://localhost:9090 --http.config.file=http-config-file.yml
+```
+
+The file is written in [YAML format](https://en.wikipedia.org/wiki/YAML), defined by the scheme described below.
+Brackets indicate that a parameter is optional. For non-list parameters the value is set to the specified default.
+
+The file is read upon every http request, such as any change in the
+configuration and the certificates is picked up immediately.
+
+Generic placeholders are defined as follows:
+
+* `<bool>`: a boolean that can take the values `true` or `false`
+* `<filename>`: a valid path to a file
+* `<secret>`: a regular string that is a secret, such as a password
+* `<string>`: a regular string
+
+A valid example file can be found [here](/documentation/examples/promtool-http-config-file.yml).
+
+## <http_config>
+`http_config` allows configuring the HTTP client that promtool uses to communicate with Prometheus.
```

Comment:
There is no `http_config` key in the file, is there? I think, these two lines should go.